homeassistant/components/sensor/co2signal.py

```diff
@@ -0,0 +1,133 @@
+"""
+Support for the CO2signal platform.
+For more details about this platform, please refer to the documentation
+"""
+import logging
+import voluptuous as vol
```

Comment:
Import should be grouped.

Reply:
Please add a blank line between standard library, 3rd party and homeassistant import groups.